contracts/TablelandTables.sol

```diff
@@ -71,6 +73,10 @@ contract TablelandTables is
             revert Unauthorized();
         }
 
+        if (bytes(statement).length > QUERY_MAX_SIZE) {
+            revert("query size greater than maximum");
```

Comment:
```suggestion
            revert("Tables: query size greater than maximum");
```

Reply:
oh, actually we could create a custom error for this... see the `Unauthorized` error in the interface